indexer/src/main/java/au/org/aodn/esindexer/service/StacCollectionMapperService.java

```diff
@@ -571,6 +571,21 @@ List<LinkModel> mapLinks(MDMetadataType source) {
         geoNetworkService.getThumbnail(this.mapUUID(source))
                 .ifPresent(results::add);
 
+        // full metadata link
+        var metadataSource = source.getMetadataLinkage();
```

Comment:
it is 100% not null for this one `source.getMetadataLinkage()`?

Reply:
i use `safeGet()` below, so null is also ok